sql/hive/src/test/scala/org/apache/spark/sql/hive/parquetSuites.scala

```diff
@@ -175,7 +175,7 @@ class ParquetMetastoreSuite extends ParquetPartitioningTest {
     (1 to 10).map(i => Tuple1(Seq(new Integer(i), null))).toDF("a")
       .createOrReplaceTempView("jt_array")
 
-    setConf(HiveUtils.CONVERT_METASTORE_PARQUET, true)
+    assert(spark.sqlContext.getConf(HiveUtils.CONVERT_METASTORE_PARQUET.key) == "true")
```

Comment:
This should no longer be needed since the flag value is true by default. I changed it to an assert to validate this.

This lets us get rid of the setConf(..., false) in the afterAll(), which was causing the conf value to be leaked to other suites.